README.md

```diff
@@ -51,6 +51,10 @@ For now, development observes the following policy for branches:
 
 `git submodule add -b master git@github.com:datajoint/pharus.git`
 
+## Building the docs
```

Comment:
I can move it for now, but probably need another following pull request with github action to actually have a link to the static hosted site

Reply:
Hmm, don't think you are following what I meant. Basically, the idea is to have the top-level `README.md` be basic user documentation and move the rest into `dev-docs`. Let me PR to you; think it will help being on the same page.